flake/dev/flake.nix

```diff
@@ -46,6 +46,11 @@
       url = "github:nix-community/home-manager";
       inputs.nixpkgs.follows = "dev-nixpkgs";
     };
+
+    spicetify-nix = {
+      url = "github:Gerg-L/spicetify-nix";
+      inputs.nixpkgs.follows = "dev-nixpkgs";
```

Comment:
This should also follow its `systems` input.

It might be better to merge this PR after https://github.com/nix-community/stylix/pull/1579 to have the architecture of commit https://github.com/nix-community/stylix/pull/1579/commits/fd95af2d6594f5e6de2f8deeb76dd6a42867ffb7 ("flake: dev: follow public flake-parts and systems inputs") available.

Reply:
> It might be better to merge this PR after #1579 to have the architecture of commit [fd95af2](https://github.com/nix-community/stylix/commit/fd95af2d6594f5e6de2f8deeb76dd6a42867ffb7) ("flake: dev: follow public flake-parts and systems inputs") available.

For reference, https://github.com/nix-community/stylix/pull/1579 has been merged.